.github/workflows/tests.yml

```diff
@@ -0,0 +1,61 @@
+name: tests
+
+on:
+  push:
+  pull_request:
+    types: [opened, synchronize, reopened, edited]
+
+jobs:
+  tests:
+    name: Test
+    strategy:
+      fail-fast: true
```

Comment:
Do we want `fail-fast: true`? I think on the other discussion I saw we wanted `fail-fast: false`.

The reason we usually do that is that it's nice to know whether the failure is build-specific, or if it's the code. In most simple bugs all the tests will be red and that's informative.

Reply:
My suggestion has been to use `false` for all the smaller repos with fast build+tests. `true` might make sense for `thinc` or `spacy`, though?

Reply:
That's on me: I used it for debug and forgot to switch it back. Will push a change soon.